app/views/build_estimates/_money_input.html.slim

```diff
@@ -0,0 +1,13 @@
+- hint_text ||= nil
+- if hint_text.present?
+  = form.govuk_number_field field,
+                              width:,
+                              hint: { text: hint_text },
+                              label: { text: label_text, size: "m" },
+                              prefix_text: BuildEstimatesHelper::POUND
+- else
+  = form.govuk_number_field field,
+                              width:,
+                              hint: { text: hint_text },
```

Comment:
I know this is a conversion, but this doesn't look quite right - if hint_text is not present we still provide it?